### DISCUSSION TITLE
Confirmation email template locally

### DISCUSSION THREAD

**Discussion by ricardosikic** [supabase/supabase#17601] [Questions]
Hi!, Is there a way to change the confirmation email url sent to inbucket when user confirms the signup? In this case the url send back the user to `localhost:3000` but i wanted to send the user back to `localhost:3000/my-account`.  I followed the guide to customize a new email template, so i added to `toml` file

```
[auth.email.template.confirmation]
subject = "App confirmation"
content_path = "./supabase/templates/confirmation.html"
```
And i created a template for that path but is not working.

**Comment by DamienDeloubes** [comment]
Want to mention, I am having this issue also 🥲

**Reply by sp88011** [reply]
Same here. Any luck anyone?

**Reply by thereis** [reply]
no luck so far!

**Reply by silentworks** [reply]
What does your `confirmation.html` content look like?

**Reply by thereis** [reply]
@silentworks even a simple html text written like `<b>{{ .Email }}</b>` is not being sent.

**Reply by silentworks** [reply]
Did you stop and start the container after making the change?

**Reply by thereis** [reply]
@silentworks yeah, I am on the latest version of the project. I get the same behaviour if I change in the cloud dashboard.

Can you also test with `recovery` as well?

**Reply by silentworks** [reply]
I've just tested this out with both `confirmation` and `recovery` with the latest version `1.163.6` of the supabase CLI. I've attached a video showing this working in my setup locally.

https://github.com/supabase/supabase/assets/79497/921804b4-a9e5-4faf-a7a5-48e956ebca87

**Reply by thereis** [reply]
@silentworks I will try replicate it here, maybe there's something wrong with my `config.toml` file.

Thank you!

**Reply by thereis** [reply]
@silentworks thank you so much, I've fixed it here by using https://github.com/maizzle/framework to generate the templates for supabase.

**Reply by vyas-meet** [reply]
> @silentworks thank you so much, I've fixed it here by using https://github.com/maizzle/framework to generate the templates for supabase.

As someone too invested in tailwind, Maizzle looks great. Is there any examples on using it with supabase? Thanks :)

**Reply by souravsspace** [reply]
> @silentworks thank you so much, I've fixed it here by using https://github.com/maizzle/framework to generate the templates for supabase.

Hey, Do you have any examples of it using supabase?

**Reply by thereis** [reply]
@souravsspace sorry but I don't, but it's pretty much compiling the application into html and saving them into the templates folder in supabase

**Comment by rindev0901** [comment]
@silentworks Can you show the template configuration code in the config.toml file? Please

**Reply by silentworks** [reply]
https://github.com/silentworks/supabase-by-example/blob/main/remix/supabase/config.toml#L128-L142
